app/presenters/queries/available_translations.rb

```diff
@@ -32,7 +32,7 @@ def web_item(id)
       end
 
       def expanded_translations
-        @expanded_translations ||= grouped_translations.map do |_, (id)|
+        @expanded_translations ||= grouped_translations.map do |_, ((id, _, _))|
```

Comment:
sure, yeah it'd be good if we could simplify it as it's a lot of brackets and underscores so quite intimidating.